scripts/ipc/src/root-single-validator.sh

```diff
@@ -11,7 +11,7 @@ sleep 5
 eudico wallet import --as-default --format=json-lotus  /scripts/ipc/src/wallet.key
 echo "[*] Initializing validator config and adding validator"
 eudico mir validator config init
-validator_addr=`eudico mir validator config validator-addr | grep -vE '(/ip6/)' | grep -E '/ip4/127.0.0.1/tcp/1347'`
+validator_addr=`eudico mir validator config validator-addr | grep -vE '(/ip6/)' | grep -E '/ip4/127.0.0.1/udp/1347/quic'`
```

Comment:
The default one is 1348: https://github.com/consensus-shipyard/lotus/blob/f081f219b9205a1ce200b0866f250048bcaec7e4/cmd/eudico/mirvalidator/cfg.go#L25 
```suggestion
validator_addr=`eudico mir validator config validator-addr | grep -vE '(/ip6/)' | grep -E '/ip4/127.0.0.1/udp/1348/quic'`
```